avalon/tools/projectmanager/widget.py

```diff
@@ -237,10 +237,13 @@ def flags(self, index):
 
     def data(self, index, role):
 
+        if not index.isValid():
+            return
+
+        node = index.internalPointer()
```

Comment:
Not too big of an issue, just note that this isn't actually **always** used. A lot of the calls to this function will pass all the way to the `super` call at the bottom.